client/scss/components/_button.scss

```diff
@@ -232,6 +232,10 @@
         color: $color-grey-3;
     }
 
+    &:not(:disabled):not(.disabled).active {
```

Comment:
I think we need a different approach for styling the 'selected' & 'non-selected'.
Currently there is only one use of this type of button (button.bicolor) and that is when you are entering tags, so this state is associated with being selected.

There is however a .button.secondary we could leverage to indicate an alternate style.

Styling changes needed
* .button.bicolor {  border: 1px solid transparent;} - this way the heights of elements do not appear different across the two states
* .button.bicolor.button-secondary {border: 1px solid #007d7e /* use variable here*/;}  - this will add a border around the tag button, otherwise it looks a bit odd.
* remove the .active class styling & the active variable below